lib/WP_Auth0_LoginManager.php

```diff
@@ -118,12 +118,6 @@ public function login_auto() {
 			return false;
 		}
 
-		// Do not redirect Auth0 login processing.
-		// TODO: This should be removed as this page is no longer used for callbacks.
-		if ( null !== $this->query_vars( 'auth0' ) ) {
```

Comment:
Callback is not actually processed on this page so this just closes a hole. 

Reason for removed tests in `tests/testLoginManager.php`.